Cesium3DTilesSelection/include/Cesium3DTilesSelection/ViewUpdateResult.h

```diff
@@ -31,6 +35,10 @@ class CESIUM3DTILESSELECTION_API ViewUpdateResult final {
    */
   std::vector<Tile*> tilesToNoLongerRenderThisFrame;
 
+  std::unordered_set<Tile*> tilesFadingOut;
+
+  std::vector<Tile*> tilesToHideThisFrame;
```

Comment:
I changed the semantics / naming of the containers in `ViewUpdateResult` and also added documentation. I think all four containers are necessary, but can you check out the new documentation that describes the usage of each one? Let me know if you still think it could be simplified.

Reply:
Not sure if this works but can we simplify to only two fields `tilesFadeInThisFrame` and `tilesFadeOutThisFrame`? `tilesFadeInThisFrame` requires the client to render the tile with alpha opacity. And `tilesFadeOutThisFrame` requires the client to render the tile with alpha opacity > 0 or hide the tile when alpha opacity < 0 and also disable the collision for fade-out tiles.

Reply:
Good idea, I applied this suggestion in the latest commit! Now the fading in tiles + the fully opaque tiles are in the `tilesToRenderThisFrame`. The fading out tiles are in `tilesFadingOut` (renaming of `tilesToNoLongerRender`). The client is responsible for hiding tiles when their `lodTransitionPercentage` is 0 - the next frame these tiles won't appear on the list, so the client needs to hide them as soon as they reach 0. 

In the case of Cesium for Unreal, the physics for tiles on the `tileFadingOut` are disabled the first time they appear on the list. Later, if they appear on the render list again, their physics meshes are reenabled.